nb/start.md

```diff
@@ -1,4 +1,4 @@
-# JSxCAD
+# Jot (JSxCAD)
```

Comment:
😮 Are we fully committing to the name change?

Reply:
Oh, we were committed already -- but I was hoping for a cleaner transition. :)